src/main/java/kafka/vertx/demo/websocket/WebsocketVerticle.java

```diff
@@ -25,7 +25,7 @@ public void start(Promise<Void> done) {
         HttpServer server = vertx.createHttpServer();
         Router router = Router.router(vertx);
         router.get().handler(StaticHandler.create());
-        server.websocketHandler(websocket -> {
+        server.webSocketHandler(websocket -> {
```

Comment:
In the latest milestone of vert.x 4.0.0 this function appears to still be spelt with a lowercase s

Reply:
This has changed in the current snapshots, and the latest milestone is back from December so currently fresh Vert.x 4 is in the snapshots.